index.html

```diff
@@ -19,9 +20,21 @@
     <div class="menu-icon-container">
       <img src="toolbar-icon.png" alt="Menu icon">
```

Comment:
- [ ] I kindly suggest that you surround your Menu icon with a `nav` tag to add [semantics](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) because it's a navigation element.